docs/CustomCommands.md

```diff
@@ -71,17 +71,17 @@ __Note:__ If you register a custom command to the `browser` scope, the command w
 browser.addCommand("myCustomBrowserCommand", function () { return 1 })
 const elem = $('body')
 console.log(typeof browser.myCustomBrowserCommand) // outputs "function"
-console.log(typeof elem.myCustomBrowserCommand()) // outputs "undefined"
+console.log(typeof elem.myCustomBrowserCommand) // outputs "undefined"
 
-browser.addCommand("myCustomElementCommand", function () { return 1 }, true)
+browser.addCommand("myCustomElementCommand2", function () { return 1 }, true)
```

Comment:
There is no `myCustomElementCommand` or `myCustomElementCommand1`, shall we start with `myCustomElementCommand1`?